pageserver/ctl/src/main.rs

```diff
@@ -219,6 +219,7 @@ fn handle_metadata(
     let mut meta = TimelineMetadata::from_bytes(&metadata_bytes)?;
```

Comment:
We no longer store metadata on disk since #6769, but good find.

Reply:
Changing via pagectl is no longer an option.

Reply:
Removed from this patch b/c now we store everything to index_part. Should we remove this functionality from pagectl?

Reply:
Let's have the cleanup on a separate PR since it's just nice code deletion, unrelated to this feature.